templates/zerver/help/include/create-bot-construct-url.md

```diff
@@ -13,3 +13,8 @@ stream name you want the notifications sent to. If you do not specify a
 If you'd like this integration to always send to a specific topic,
 just include the (URL-encoded) topic as an additional parameter
 (E.g. for `your topic`, append `&topic=your%20topic` to the URL).
+
+{% if all_event_types is defined %}
+{!event-filtering-instruction.md!}
+
+{% endif %}
```

Comment:
This is a nit, but I'd like the spacing here to be consistent. So either no empty lines or empty lines between each, unless there is a reason I'm not aware of to have it formatted this way.

A. 
> {% if all_event_types is defined %}
> {!event-filtering-instruction.md!}
> {% endif %}

B.
> {% if all_event_types is defined %}
>
> {!event-filtering-instruction.md!}
>
> {% endif %}

Reply:
I think we should go with B because we do that in other files.